### DISCUSSION TITLE
Team meeting planning discussion for 2022-02-23

### DISCUSSION THREAD

**Discussion by nalininsbs** [wpmarketingteam/Marcomms-Internal-Comms#16] [General]
Please add any input into this discussion.

**Comment by murillol** [comment]
Hi @nalininsbs @eidolonnight, below is the Agenda outline. 

I've included the Polyglots newsletter because I understand it will be published by Wednesday. Also including the coffee break that will take place. The Call for WCEU Volunteers could go as an item on Open Floor?

One item I think would be nice to promote consistently (in general, not necessarily this week) are the FSE Calls for Testing and Hallway Hangouts. 

Regarding Notes, Ugyen mentioned he is unwell so I've reached out to ask him if he is still OK to do the Notes from the last meeting. Haven't received a response, so if he can't I can do them. 

**Agenda**: Global Marketing Team, 23 February 2022

Join the team at 15:00 UTC in the [Marketing channel on the Make WordPress Slack](https://wordpress.slack.com/archives/C0GKJ7TFA). Async contribution is welcome too.

1. **Welcome**
2. **Getting to know the team and participating**
    a. Attendees
    b. Social question of the week
3. Notes from the Last Meeting
    (Link)
    Volunteer call for notetakers.
 4. **Project Updates**
    a. Polyglots Newsletter
    b. Social media
    c. Learn WordPress
    d. GitHub updates and links
    e. [#WPDiversity](https://make.wordpress.org/marketing/tag/wpdiversity/)
5. **Diary/Calendar Dates**
    Follow this Google [Calendar with meetings, collaborations, and social activities](https://calendar.google.com/calendar/b/2?cid=NDQyN2g1cnZrN2EwYXA5Y3VpNDI5OGYzY3NAZ3JvdXAuY2FsZW5kYXIuZ29vZ2xlLmNvbQ). We also list important dates in this [Forthcoming Diary Dates](https://make.wordpress.org/marketing/2021/01/13/forthcoming-diary-dates/) post. Join this week's coffee break at 6.30pm UTC. Link will be shared on Slack.
6. Open Floor

If you have any additions in mind, please comment below. We will see you Wednesday!

**Reply by nalininsbs** [reply]
Thank you Larissa. I will read through it again tomorrow. We have been without power for part of the day with the storm. I have made some minor changes.

**Reply by nalininsbs** [reply]
I agree on FSE

**Reply by eidolonnight** [reply]
Checking with @DanSoschin to see if we need a recurring spot for Learn or if this should be bundled into the social media item.

**Comment by murillol** [comment]
**Notes Update**: Ugyen has submitted his Notes draft on the P2 and I've looked through it, reformatted and added links. Can one (or both) of you give it a final lookover and we can publish? Thanks @nalininsbs @eidolonnight !

**Comment by eidolonnight** [comment]
Script for meeting started: https://docs.google.com/document/d/1tIkufElGGR_PQOlJzscyFw_2F0OdP4evrntjlm1F15Q/edit#

**Reply by murillol** [reply]
I added:
- WP Diversity item
- In the Diary Dates section we should address the question about daylight savings: _will meetings continue at 15:00 UTC after daylight savings?_
- A bit on the Polylgots Newsletter
- @nalininsbs what will the social question be? :)

**Reply by nalininsbs** [reply]
I can do polyglots newsletter update as we had a change on video interview.

I have set up the discussion post for next week's meeting ready too so we can link to that during the meeting.

Social question: What is the book you would recommend to others and why?

Can we also thank everyone who came and sent messages to the celebration last week. I will write up a post once those away or ill have sent their message in as it will encourage others. But need the storms to be over so have full electric. 

Thank you @murillol

**Reply by nalininsbs** [reply]
I will aim to add into script in case power and internet not good tomorrow during meeting. One more major storm to come later this week.

**Reply by nalininsbs** [reply]
we should link to 5.9.1 from today too. I can mention that if needed .

**Comment by murillol** [comment]
hi @nalininsbs @eidolonnight 

**Script for tomorrow**

-  I've filled in a bit for #WPDiversity explaining the new promotion process going forward (and from now on I suppose it can be bundled in with the Social Media item?)
- I've included a placeholder about Daylight Savings in the Diary Dates section

**Agenda for tomorrow**
- I left a draft of the Agenda in the P2
- Waiting to see if Learn WP is still its own separate item
- Waiting to see if we have any decision on the Daylight Savings thing so we can include it

See you tomorrow!

**Reply by nalininsbs** [reply]
Is there some review together still needed of this new promotion process @murillol unless I am missing some of the replies on this? [relating to WPDiversity] Thank you

**Reply by murillol** [reply]
So Jill has been asked to coordinate items directly with Anjana from now on: https://wordpress.slack.com/archives/C0GKJ7TFA/p1645025309109359 - since there is a documented process that is at odds with this newer process, I think it's a good idea we clarify on the WPDiversity item in the meeting today @nalininsbs

**Reply by nalininsbs** [reply]
Yes I agree but it is important for its context and this needs further discussion too.

**Reply by murillol** [reply]
I'm not sure I understand @nalininsbs - what is important for context?

**Comment by jpantani** [comment]
Hello @eidolonnight, @nalininsbs, and @murillol 

I have completed the draft of the notes for 22 February 2022 in the Google doc: https://docs.google.com/document/d/1tIkufElGGR_PQOlJzscyFw_2F0OdP4evrntjlm1F15Q/edit#

Of note, I am proposing one header change for section 2. I am recommending language that removes some psychological barriers to being new "Getting to know each other." Alternatively, we can say "Meet the team".

Also in the meeting, Section 6 was intentionally skipped so noted it. I don't want the meeting notes to seem like there's an error. 

I have prepared a P2 draft of the notes and will await feedback before publishing. (https://make.wordpress.org/marketing/wp-admin/post.php?post=8772&action=edit)

Thank you for your collaboration!

**Reply by nalininsbs** [reply]
Thank you @jpantani . I have reviewed and made some minor edits. I have made most of these on the blog draft too for time and as internet access may be challenging in the week ahead. I have two queries and once @eidolonnight and @murillol have chance to reply, these can be clarified in the notes. We should be ready to publish after that.

Thank you for your suggestion of 'Getting to know each other' and your message about this in the google document. I agree that this sounds more inclusive. I prefer this to 'meet the team'. Larissa and Nicholas, if you agree, we can use it for the future agenda sub-headings too.

**Reply by murillol** [reply]
Thanks @jpantani and @nalininsbs I  left my answers to these items in the document:

- UTC meeting time - I think what @nalininsbs added is good
- Update a section of agenda to "Getting to know each other"?  - I agree with this change
- Do we need to put in the number of meeting attendees? - no, probably not necessary to include them
